packages/env/lib/create-docker-compose-config.js

```diff
@@ -0,0 +1,55 @@
+module.exports = function createDockerComposeConfig(
+	pluginPath,
+	pluginName,
+	pluginTestsPath
+) {
+	const commonVolumes = `
+      - ${ pluginPath }/:/var/www/html/wp-content/plugins/${ pluginName }/
+      - ${ pluginPath }${ pluginTestsPath }/e2e-tests/mu-plugins/:/var/www/html/wp-content/mu-plugins/
+      - ${ pluginPath }${ pluginTestsPath }/e2e-tests/plugins/:/var/www/html/wp-content/plugins/${ pluginName }-test-plugins/`;
+	const volumes = `
+      - ${ pluginPath }/../${ pluginName }-wordpress/:/var/www/html/${ commonVolumes }`;
+	const testsVolumes = `
+      - tests-wordpress:/var/www/html/${ commonVolumes }`;
+	return `version: '2'
+volumes:
+  tests-wordpress:
+services:
+  mysql:
+    environment:
+      MYSQL_ROOT_PASSWORD: password
+    image: mariadb
+  wordpress:
+    depends_on:
+      - mysql
+    environment:
+      WORDPRESS_DEBUG: 1cq
+      WORDPRESS_DB_NAME: wordpress
+      WORDPRESS_DB_PASSWORD: password
+    image: wordpress
+    ports:
+      - 8888:80
```

Comment:
Hard-coding the port numbers can be problematic, as there will always be clashes with other things. Allowing it to be customised through environment variables is a pretty reliable method.

Reply:
How about `--port` and `--tests-port` options? They're more self documenting.

Reply:
I suspect that folks would want a way to permanently set the port, so they don't have to remember to add an option every time. It's something that `docker-composer.override.yml` was used for before the switch to the `@wordpress/scripts` method.

Reply:
That makes sense, I'll go with env vars then.